pdf2image/pdf2image.py

```diff
@@ -9,6 +9,7 @@
 import uuid
 import tempfile
 import shutil
+import pathlib
```

Comment:
That would work, but is it worth it?
You might as well just create a separate branch for 2.7 and avoid touching it until the support for that version is dropped.

Reply:
It is indeed not worth it, I thought `pathlib` was not supported by 3.5, I do not care much for 2.7 since the EOL is in 4 months.

I'll mark add a disclaimer that the next release drops python 2 support and that will be the end of it.